planning/obstacle_avoidance_planner/include/obstacle_avoidance_planner/mpt_optimizer.hpp

```diff
@@ -128,6 +128,7 @@ struct ReferencePoint
   //       second is fixing current ego pose when no velocity for planning from ego pose
   boost::optional<Eigen::Vector2d> fix_kinematic_state = boost::none;
   bool plan_from_ego = false;
+  double max_plan_from_ego_length = 10.0;
```

Comment:
This is not required in ReferencePoint.

Reply:
Remove it https://github.com/autowarefoundation/autoware.universe/pull/1191/commits/8a03c11cfc83d9870d6ec8f3388ffa9fee5890e8